reporter/html.js

```diff
@@ -707,8 +710,18 @@ QUnit.testDone(function( details ) {
 		}
 	}
 
+	if (config.collapse !== collapseCheck) {
+		collapse = config.collapse;
+	}
```

Comment:
as already mentioned by Jörn, it's not necessary to make this variable copy. You can remove all this block and the `collape` declaration on the top of the file.